src/AlgAss/AlgQuat.jl

```diff
@@ -1,3 +1,18 @@
+@doc raw"""
+    quaternion_algebra(K::Field, a, b) -> QuaternionAlgebra
+
+Return the quaternion algebra $(a, b | K)$ defined by $i^2 = -1$, $j^2 - 1$.
```

Comment:
Do you mean :
```suggestion
Return the quaternion algebra $(a, b | K)$ defined by $i^2 = a$, $j^2 = b$.
```
?

Reply:
yeah sounds better (although always taking -1 would make everything easier)